src/Standard.php

```diff
@@ -18,14 +18,36 @@ class Standard
      */
     public static function loadFromXmlFile(string $file): self
     {
-        if (!file_exists($file)) {
-            throw new \InvalidArgumentException(sprintf('Cannot find standards file "%s".', $file));
+        if (file_exists($file)) {
+            $standard_file = $file;
+        } else {
+            // Is it a default standard?
+            $standard_file = __DIR__ . '/Standard/' . $file . '.xml';
```

Comment:
Should we verify whether there might be stuff like `../` present in `$file`?

Reply:
does it matter? Those paths are handled fine with `file_get_contents`.

Reply:
Might be a security problem, but it is data provided by the user and not from the web, so I see no bug but a feature.

Reply:
well line 25 is there when you want to "include" the standard "Hostnet". I could check if file contains `[a-zA-Z_-0-9]+`.

Reply:
Line 25 indeed.